docs/docs-ref-autogen/excel/excelscript/excelscript.autofilter.yml

```diff
@@ -6,7 +6,29 @@ fullName: ExcelScript.AutoFilter
 summary: >-
   Represents the `AutoFilter` object. AutoFilter turns the values in Excel column into specific filters based on the
   cell contents.
-remarks: ''
+remarks: |-
+
+
+  #### Examples
+
+  ```TypeScript
+  /**
+   * This script creates an autofilter on the worksheet that filters out rows based on column values. 
+   * The autofilter filters to only include rows that have a value in column C in the lowest 10 values 
+   * (of column C values).
+   */
+  function main(workbook: ExcelScript.Workbook) {
+    const currentSheet = workbook.getActiveWorksheet();
+    const dataRange = currentSheet.getUsedRange();
+
+    // Add a filter that will only show the rows with the top 10% of values in column C
```

Comment:
This says top 10% of values, but the preceding description says lowest 10 values, and it's using bottomItems. Should this say lowest 10% of values? Or lowest 10 values?

Reply:
The sample filters to the 10 lowest valued items. I'll update this comment accordingly.